### ISSUE TITLE
float deviation make chart look ugly

### ISSUE THREAD

**Issue by immofon** [marianogappa/chart#15]
input csv data:
```csv
0,0.604660,0.940509
1,0.664560,0.437714
2,0.424637,0.686823
3,0.065637,0.156519
4,0.096970,0.300912
5,0.515213,0.813640
6,0.214264,0.380657
7,0.318058,0.468890
8,0.283034,0.293102
9,0.679085,0.218553
```
cmd: `cat data.csv | chart line , `

got:
![screenshot from 2017-07-01 16 11 08](https://user-images.githubusercontent.com/16877933/27760350-3a74eb72-5e0b-11e7-828a-9deab6c22a5d.png)

**Comment by marianogappa** [comment]
I'm afraid this is a problem with the `Chart.js` library. Something similar to this: https://github.com/chartjs/Chart.js/issues/1890

I've just updated the `Chart.js` code to the current latest version (2.6.0) hoping the problem would have been solved but sadly it's not.

Unfortunately, `chart` ignores how `Chart.js` will decide to display the y-axis, and preemptively rounding y-axis values when they might not cause a problem sounds inefficient and undesired behaviour.

I'll have to think about this a little bit more. Suggestions are appreciated!
